subprojects/gstreamer/gst/gstelement.c

```diff
@@ -3633,7 +3638,7 @@ gst_element_get_contexts (GstElement * element)
   g_return_val_if_fail (GST_IS_ELEMENT (element), NULL);
 
   GST_OBJECT_LOCK (element);
-  ret = g_list_copy_deep (element->contexts, (GCopyFunc) gst_context_ref, NULL);
+  ret = g_list_copy_deep (element->contexts, gst_context_gcopy_proxy, NULL);
```

Comment:
What do you think about creating a function in gstcontext.c/.h named gst_context_ref_with_data?

Reply:
I think not here, otherwise we have to fix several stuff. So I just approve.